index.bs

```diff
@@ -150,7 +196,20 @@ the following steps:
 1. Return |encodedOutput|.
 
 <h3 algorithm id="delivery-time">Establishing report delivery time</h3>
-TODO
+To <dfn>establish report delivery time</dfn> given an [=impression=] and a
+[=conversion/conversion time=], run the following steps:
+1. Let |timeDelta| be the difference between the [=conversion/conversion time=] and [=impression/impression time=].
+2. Let |expiryDelta| be the difference between the [=impression/expiry=] and the [=impression/impression time=]
+1. If |timeDelta| <= (2 days - 1 hour), return [=impression/impression time=] + 2 days
```

Comment:
Consider using https://w3c.github.io/tr-design/src/readme.html#switch to improve the formatting for this block.